parser/src/main/java/com/microsoft/gctoolkit/parser/jvm/JVMConfiguration.java

```diff
@@ -6,6 +6,8 @@
 
 public interface JVMConfiguration {
 
+    int MAXIMUM_LINES_TO_EXAMINE = 10000;
```

Comment:
My old eyes find 10_000 much easier to read.

Reply:
done...